hugo/content/_index.md

```diff
@@ -64,6 +64,11 @@ pgMonitor combines multiple open-source software packages and necessary configur
 
 - PostgreSQL 10+ SCRAM-SHA-256 encrypted passwords are supported on the Linux version of pgMonitor 4.0 or later only.
 
+### PgBouncer
+
+- PgBouncer 1.17
```

Comment:
There was another reference in a different PR for PgBouncer 1.19 being the required version; any impact here?

Reply:
Happen to recall where that was? We did had a previous requirement of at least 1.18 in the last pgbouncer_fdw version. This backported the support because I needed to be able to test a version difference that mattered.

Reply:
I think I might be confusing w/HA pgbouncer requirement bump.